components/NotificationBar.js

```diff
@@ -1,21 +1,54 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
+import styled from 'styled-components';
+
+import AppRejectionReasonModal from '../components/host-dashboard/AppRejectionReasonModal';
+import AcceptReject from '../components/host-dashboard/AcceptReject';
 
 const logo = '/static/images/opencollective-icon.svg';
 
+const AcceptReject1 = styled.div`
```

Comment:
Adding `1` at the end is not a clean way of differentiating these variables. Could you provide a better name?

Reply:
Done